src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+  const stylesArray = sourceString.split(';').filter(Boolean);
+
+  stylesArray.forEach((style) => {
+    const parts = style.split(':');
+
+    if (parts.length < 2) {
+      return;
+    }
+
+    const property = parts[0].trim();
+    const value = parts[1].trim();
+
+    if (property && value) {
+      stylesObject[property] = value;
+    }
```

Comment:
[CODE STYLE] - Avoid unnecessary conditionals. Since you're already trimming `parts[0]` and `parts[1]`, and considering that CSS properties and values are non-empty strings, you can directly assign them to `stylesObject` without the `if` condition.